docs/source/find_data/stations/coimbra.md

```diff
@@ -44,7 +44,7 @@ thermal paper
 In 2003, data began being recorded and stored digitally. The last helicorder data was recorded in 2006.
 
 ## Data Availability
-Some seismograms have been digitized and can be found through EUROSEISMOS [website](http://storing.ingv.it/es_web/).
+Some seismograms have been digitized and can be found through EUROSEISMOS [website](https://storing.ingv.it/es_web/index.htm). <!--broken link update kindly verify-->
```

Comment:
This is good.

The style preference for pull requests is that you DO NOT put an html comment in (as becomes part of the repo) but instead put this in the comment of the pull request.  That way I can simply accept the change and not ask uyou to remove the comment.

Please remove the html comment.